src/client/pages/OfferNew/Introduction/Sidebar/StartDate.tsx

```diff
@@ -313,10 +254,15 @@ const DateForm: React.FC<{
         size={size}
       >
         <Value>
-          <DateLabel hasStartDate={Boolean(dateValue)}>
-            {getDateLabel()}
+          <DateLabel>
+            {!hasStartDate && quote.dataCollectionId && (
+              <StartDateLabelSwitcher
+                dataCollectionId={quote.dataCollectionId}
+              />
+            )}
+            {hasStartDate && getDateLabel()}
           </DateLabel>
-          {hasStartDate && <DownArrow />}
+          <DownArrow />
```

Comment:
So we hid `DownArrow` before if it was a Switcher? Don't understand why so I guess it's correct to remove it 😄

Reply:
Ah yes - sorry should have added that to the description of the PR! 🙈 But yes, I think it makes sense to indicate that you can still pick another date. And without the arrow that's _really_ unclear.

Reply:
So I removed that conditional.